cli/src/main/java/keywhiz/cli/configs/UndeleteActionConfig.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package keywhiz.cli.configs;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+@Parameters(commandDescription = "Undelete a secret that has previously been soft-deleted")
+public class UndeleteActionConfig {
+
+  @Parameter(description = "<secret>")
+  public String objectType;
+
+  @Parameter(names = "--id", description = "ID of the item to delete", required = true)
```

Comment:
Depending on whether this is truly secret-specific or not, we may want to be consistent in terminology between lines 22 and 28 (undelete secret/id of secret, etc.).